src/NServiceBus.PersistenceTests/Sagas/When_persisting_a_saga_with_an_escalated_DTC_transaction.cs

```diff
@@ -34,14 +35,12 @@ public async Task Save_should_fail_when_data_changes_between_concurrent_instance
                         using (var unenlistedSession = configuration.CreateStorageSession())
                         {
                             await unenlistedSession.Open(unenlistedContextBag);
-
                             await enlistedSession.TryOpen(transportTransaction, enlistedContextBag);
 
                             var unenlistedRecord = await persister.Get<TestSagaData>(generatedSagaId, unenlistedSession,
                                 unenlistedContextBag);
```

Comment:
While we're in there, let's go ahead and get rid of the line break here as well.